pegasus/emails/2017_teacher_application_accept_teachercon.md

```diff
@@ -4,7 +4,7 @@ subject: "Congratulations from Code.org! Respond by April 21st"
 ---
 Hi <%= preferred_first_name_s %>,
 
-We are so excited to offer you a space in Code.org’s Professional Learning Program for <%= course_name_s %>! We think you can make an important contribution to our collective goal to ensure computer science education is available to every student in every school across the country, and we can’t wait to start working with you. Please read through the following information, and then make sure to let us know your plans via the form below.
+Code.org and <%= regional_partner_name_s %> are so excited to offer you a space in Code.org’s Professional Learning Program for <%= course_name_s %>! We think you can make an important contribution to our collective goal to ensure computer science education is available to every student in every school across the country, and we can’t wait to start working with you. Please read through the following information, and then make sure to let us know your plans via the form below.
```

Comment:
Are we guaranteed that regional_partner_name_s is non-null? If it's possible that the RP is unset here then I'd suggest making this a ternary OR throwing an exception before calling the template